src/auth/auth-api-request.ts

```diff
@@ -401,6 +509,51 @@ export class FirebaseAuthRequestHandler {
         });
   }
 
+  /**
+   * Imports the list of users provided to Firebase Auth users database. This is useful when
```

Comment:
Lets drop `database` from the text. Here and elsewhere.

Reply:
done